.clang-format

```diff
@@ -24,5 +24,7 @@ ObjCSpaceBeforeProtocolList: false
 PenaltyBreakBeforeFirstCallParameter: 1
 PenaltyReturnTypeOnItsOwnLine: 300
 PointerAlignment: Left
+SpacesInContainerLiterals: false
```

Comment:
This change doesn’t seem to have anything to do with the original PR but is instead changing our syntax style?

Reply:
These rules are consistent with the llvm template for clang-7 but not for clang-9. So basically its overwriting some rules to make them behave like clang-7 for other versions.